ui/Screen/Funding/Pool/Outgoing.svelte

```diff
@@ -5,14 +5,22 @@
   import Support from "./Outgoing/Support.svelte";
 
   import * as fundingPool from "../../../src/funding/pool";
+  import * as transaction from "../../../src/transaction";
 
   export let pool: fundingPool.Pool;
+
+  let ongoingOnboardingTx = false;
+  transaction.store.subscribe(_ => {
```

Comment:
We should avoid using `.subscribe` and use reactive statements (`$:`) instead. Explicit ubscriptions will need to be cleaned up or they leak memory and computation otherwise.

Reply:
Done :)